respondd.py

```diff
@@ -38,6 +38,7 @@ def handle(self):
             # Clone global environment and populate with interface-specific data
             local_env = dict(env)
             local_env['batadv_dev'] = batadv_dev
+            local_env['domain_code'] = domain_code
```

Comment:
Okay, I probably misunderstood how the local_env is used exactly.

Reply:
Well, `local_env` is intended for values that can change per request. In contrast the global environment - which was hard to spot since it is only passed as a function argument - contains values that are fixed, no matter what interface a request is received on.